doc/api/os.md

```diff
@@ -1160,6 +1160,38 @@ The following error codes are specific to the Windows operating system:
   </tr>
 </table>
 
+### dlopen Constants
+
+The following signal constants are exported by `os.constants.dlopen`.
+See `man 3 dlopen` for detailed informatio.
+
+<table>
+  <tr>
+    <th>Constant</th>
+  </tr>
+  <tr>
+    <td><code>RTLD_LAZY</code></td>
```

Comment:
If the idea is to refer the user to a manual, then having these as links to that manual would be ideal :-) ... still, a very brief one sentence description would be preferred (I know we're fairly inconsistent with that in the docs and we need to get better at it)